frontend/src/components/AddElection.tsx

```diff
@@ -34,8 +40,30 @@ const AddElection = ( {  }) => {
             alert('Please add election name')
             return
         }
-        console.log({ElectionName: electionName,Candidates:candidateNames})
-        onAddElection({ElectionName: electionName,Candidates:candidateNames})
+        const NewPoll : Poll = {
+            pollId: '0',
+            title: electionName,
+            candidates: candidateNames.map( (name,ind) => {
+                return {
+                    candidateId:   String(ind),
+                    shortName:     name, // short mnemonic for the candidate
+                    fullName:      name,
+                    } as Candidate;
+            })
+        }
+
+        const NewElection : Election = {
+            electionId:    '0', // identifier assigned by the system
+            frontendUrl:   '', // base URL for the frontend
+            title:         electionName, // one-line election title
+            description:  '', // mark-up text describing the election
+            startUtc:     new Date(),   // when the election starts 
```

Comment:
I added an issue to support these fields in the UI, I think that would be a good first issue for a newcomer :)

https://github.com/Equal-Vote/star-server/issues/24